guides/anchor.md

```diff
@@ -354,7 +354,7 @@ You should get a response like:
 
 ## Compliance
 
-The final piece of the puzzle is handling regulatory compliance, like <abbr title="Know Your Customer">KYC</abbr>. The [Stellar compliance protocol](compliance-protocol.md) is a standard way to exchange compliance information and pre-approve a transaction with another financial institution.
+The final piece of the puzzle is handling regulatory compliance, like <abbr title="Anti-Money Laundering">AML</abbr>. The [Stellar compliance protocol](compliance-protocol.md) is a standard way to exchange compliance information and pre-approve a transaction with another financial institution.
```

Comment:
Ah, gotcha. I had thought KYC referred to exchanging the information as well, not just initially gathering it. Will keep that in mind!